### ISSUE TITLE
SA1127 (GenericTypeConstraintsMustBeOnOwnLine) doesn't reported for parameters on multiple lines

### ISSUE THREAD

**Issue by Maxwe11** [DotNetAnalyzers/StyleCopAnalyzers#1476]
```c#
private void Foo<T>(
            int a,
            int b) where T : IFormattable
{
}
```
**Actual:** no report
**Expected**: SA1127

**Comment by Maxwe11** [comment]
interesting
```c#
[Obsolete]
private void Foo<T>(int a, int b) where T : IFormattable
{
}
```
**Actual:** no report
**Expected**: SA1127

**Comment by sharwell** [comment]
Good catches!

I think I have these fixed... final tests running now.
